game.py

```diff
@@ -1,5 +1,5 @@
 """
-Platformer Game
+Bagel Queen
```

Comment:
this is ok for now but we are going to need to change this when the game isn't about bagels

Reply:
thank you. I am not sure why we would ever make the game anything besides bagels, but that's fine